pkg/networkservice/connectioncontextkernel/ethernetcontext/setmacaddress/server.go

```diff
@@ -14,11 +16,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package macaddress provides networkservice chain elements for setting the mac address on kernel interfaces
-package macaddress
+// Package setmacaddress provides networkservice chain elements for setting the mac address on kernel interfaces
+package setmacaddress
```

Comment:
Question, why did you rename this setmacaddress ?

Reply:
> Question, why did you rename this setmacaddress ?

I did rename this for name convention.
The next step is to add `getmacaddress` package https://github.com/networkservicemesh/sdk-vppagent/pull/38#discussion_r370341434. I guess it should be consistent with `macaddress` package.

Reply:
Lets leave this as macaddress.  It follows the convention *other* places in the code so far.  getmac is sort of an odd outlier itself, as we don't do that for most things.

Reply:
By `getmac` I mean the chain element described in https://github.com/networkservicemesh/sdk-vppagent/issues/27.

As I understand this element should find created by vpp-agent mac address of interface and put it into ethernetcontext.DstMac. And it should be useful for forwarders implementation.
Did I miss something?